friends/src/components/FriendsComponents/FriendsList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import PropTypes from "prop-types"
+
+import { FriendsListContainer, Preview } from './FriendsStyleComponents'
+// import Form from '../SharedComponents/Form'
+// import Friend from './Friend'
+
+const FriendsList = props => {
+  const { friends } = props
+  return (
+    <FriendsListContainer>
+      <h1>Friends of Lambda School</h1>
+      {friends.length > 0 ? (friends.map(friend => (
+          <Link key={friend.id} to={`/friends/${friend.id}`}>
+            <Preview>
+              <div>{friend.name}</div>
+              <div>id: {friend.id}</div>
+            </Preview>
+          </Link>
+        ))) : (
+          <h2>Loading...</h2>
```

Comment:
I like that you have this conditional render for the friends list based on its length! In a more full-fledged application, it would be preferable to differentiate between an empty friends list in a loading state and an empty friends list that has finished retrieval and found nothing there. Redux helps with this differentiation :)